dali/test/python/test_dali_cpu_only.py

```diff
@@ -978,6 +978,24 @@ def test_tensor_subscript():
 def test_subscript_dim_check():
     check_single_input(fn.subscript_dim_check, num_subscripts=3)
 
+
+def test_file_properties():
+    @pipeline_def
+    def file_properties(root_path):
+        read, _ = fn.readers.file(file_root=root_path)
+        return fn.get_property(read, key="source_info")
+
+    root_path = os.path.join(data_root, 'db', 'single', 'png')
+    ref_paths = ["0/cat-1046544_640.png", "0/cat-111793_640.png", "0/cat-1245673_640.png", "0/cat-2184682_640.png",
+                 "0/cat-300572_640.png", "0/cat-3113513_640.png", "0/cat-3449999_640.png", "0/cat-3504008_640.png"]
+    p = file_properties(root_path, batch_size=8, num_threads=4, device_id=None)
+    p.build()
+    output = p.run()
+    for out in output:
+        for source_info, ref in zip(out, ref_paths):
+            assert np.array(source_info).tobytes().decode() == ref
+
+
```

Comment:
seems like this is not used. Remove?

Reply:
I'm not sure what you mean. It's a CPU-only test for `fn.get_property`.